src/main/resources/db/changelog/alert/4.0.0/changelog-authentication.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
+    <changeSet author="psantos" id="1543502711058-1">
+        <createTable tableName="ROLES" schemaName="ALERT">
+            <column autoIncrement="true" name="ID" type="BIGINT(19)">
+                <constraints primaryKey="true" primaryKeyName="ROLE_KEY"/>
+            </column>
+            <column name="ROLENAME" type="VARCHAR(255)"/>
+        </createTable>
+    </changeSet>
+    <changeSet author="psantos" id="1543502711058-2">
+        <createTable tableName="USERS" schemaName="ALERT">
+            <column autoIncrement="true" name="ID" type="BIGINT(19)">
+                <constraints primaryKey="true" primaryKeyName="USER_KEY"/>
+            </column>
+            <column name="USERNAME" type="VARCHAR(255)">
+                <constraints foreignKeyName="USER_NAME_FOREIGN_KEY" unique="true" uniqueConstraintName="CONSTRAINT_USER_NAME_UNIQUE"/>
+            </column>
+            <column name="PASSWORD" type="VARCHAR(255)"/>
+        </createTable>
+    </changeSet>
+    <changeSet author="psantos" id="1543502711058-3">
+        <createTable tableName="USER_ROLES" schemaName="ALERT">
```

Comment:
Each of these columns is a foreign key to a different table, we could add a "<addForeignKeyConstraint /> after creation to keep them linked.
Example: 
`
<addForeignKeyConstraint
                baseTableSchemaName="ALERT"
                baseTableName="USER_ROLES"
                baseColumnNames="USER_ID"
                constraintName="FK_USER_ID"
                referencedTableSchemaName="ALERT"
                referencedTableName="USERS"
                referencedColumnNames="ID"
                onDelete="CASCADE"
        />
`